labcontrol/db/tests/test_process.py

```diff
@@ -1988,12 +2018,6 @@ def test_generate_sample_sheet_shotgun(self):
         exp = SHOTGUN_SAMPLE_SHEET.format(date=tester_date)
         self.assertEqual(obs, exp)
 
-    def test_generate_sample_sheet_unrecognized_assay_type(self):
```

Comment:
Is this test replaced by any other tests due to refactoring, or is it now completely obsolete?

Reply:
It is obsolete, yes. Please review the code in sheets.py and the original issues as mentioned above. In brief, the objective of this change was to remove assay_type as a property of SequencingProcess and make it a property of PoolComposition. Hence this test no longer makes sense here.